### DISCUSSION TITLE
Save Conversation Context in VS Code Extension for Pieces Copilot

### DISCUSSION THREAD

**Discussion by anchapin** [pieces-app/support#320] [Ideas]
I added a few folders to conversation context but when I restarted my Macbook Pro and went back into VS Code, in the same repo, I don't see those folders anymore. It would be nice if the context could be saved per repository/directory on your computer.

**Comment by hra42** [comment]
you mean you don't see the folders as current context or you don't see the folders at all on you mbp?

**Reply by anchapin** [reply]
The former. The folders still are on my laptop's hard drive, but they are no longer linked to the conversation context in Pieces extension of VS Code.
